src/ui/components/shared/UserSettingsModal/PreferencesSettings.tsx

```diff
@@ -151,7 +152,7 @@ function UiPreferences() {
           />
         </div>
       </div>
-      {hitCountsEnabled && (
+      {/* {hitCountsEnabled && (
```

Comment:
Couldn't we just wire this setting up to the new one index db store? Not sure I understand why we're _removing_ this vs just connecting the two

Reply:
This is a totally separate persistence problem.

The issue here is that we used to store a "hit counts mode" enum (on/color-bar-only/show-numbers), in `localStorage`, using the legacy FF prefs system.

When you rewrote the source viewer, it got switched to a single boolean, in `localStorage`, via the `useLocalStorage` hook.

That broke this checkbox.  It still toggles the FF-based prefs value, but the source viewer code never looks at that value.

So, this checkbox is broken, and I'm hiding it from the UI until we get around to making it work again.

Reply:
Sure sure.  I was just asking if we couldn't keep this check box around, but wire it up to the new `localStorage` boolean instead. If that's hard for some reason bc of how the prefs settings dialog is wired up, that's cool. I was just wondering.

Reply:
Mmmm... couple thoughts. One is that I mis-spoke - it's not actually a "checkbox", it's a dropdown for the enum.

We _could_ theoretically change this to target `useLocalStorage`, but I also don't know if the source line rendering logic is set up to allow the 3 separate options (no hit display, color bar only, show numbers).  So, we'd either have to switch this to be a checkbox, or update the rendering logic to match prior 3-mode behavior.  Either way I'd rather do that in a follow-up PR, and there's an open issue for it.